solidity/contracts/Zoltar.sol

```diff
@@ -170,20 +169,19 @@ contract Zoltar {
 		universe.forkTime = block.timestamp;
 		universes[_universeId] = universe;
 
-		migrateREPInternal(_universeId, REP_BOND, marketResolutionData.outcome, marketResolutionData.initialReporter, marketResolutionData.initialReporter);
-		migrateREPInternal(_universeId, disputeStake, _outcome, msg.sender, msg.sender);
+		migrateREPInternal(_universeId, REP_BOND, marketResolutionData.initialReporter, marketResolutionData.initialReporter, marketResolutionData.outcome);
+		migrateREPInternal(_universeId, disputeStake, msg.sender, msg.sender, _outcome);
 	}
 
-	function migrateREP(uint192 universeId, uint256 amount, uint8 outcome) public {
-		migrateREPInternal(universeId, amount, outcome, msg.sender, msg.sender);
+	function migrateREP(uint192 universeId) public {
```

Comment:
I wonder if we should rename this to `UnwrapForkedRep` to be a bit more clear that it is not migrating anything, but instead turning your wrapped legacy REP into REP in each outcome?